static/sass/styles.scss

```diff
@@ -1143,3 +1143,38 @@ hr.p-separator.is-shallow {
 .no-border::after {
   content: none !important;
 }
+
+// silicon custom styles
+.p-heading--silicon-bold {
+  font-weight: $font-weight-bold;
+}
+
+.p-heading--silicon-light {
+  font-weight: $font-weight-display-heading;
+}
+
+.p-logo-section__separator {
+  margin-left: 2rem;
+
+  @media (max-width: $breakpoint-small) {
+    margin-left: 1.5rem;
+  }
+}
+
+.p-card--silicon-h-suru {
+  align-items: center;
+  aspect-ratio: 269/150;
+  background: #000 url("https://assets.ubuntu.com/v1/2923651d-background-1.png");
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.p-card--silicon-v-suru {
+  @extend .p-card--silicon-h-suru;
+
+  aspect-ratio: 253/422;
```

Comment:
the ratio here is 2x3 and it should be via a reusable class, instead of puting the pixel dimentions in as ratio.